### PR TITLE
Updates Hedgehog Icon, adds Emag effect

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # 

:cl: Spankmaster
add: Adds emag effect which turns a hedgehog blue upon emagging
tweak: Fixed hedgehog icon being in different locations on the tile depending on which direction it's facing
/:cl:

[why]: # To fix hedgehog

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
lame

the emag effect could have been a lot more than just making it sonic, simply hitting it with red shoes would have done the emag thing for you

also what's with you and emagging animals?

**Comment by ghost** [comment]
Give me a better idea and I'll rewrite it. The guy who sprited the hedgehog sent in a sonic sprite as well so I wanted to at least do something with it and added the first thing I could think of.

**Comment by Dinopeaks** [comment]
make it player controlled and it can attack random people for like 10 dmg  and run fast as fuck?

**Comment by ghost** [comment]
Not a bad idea Dino

**Comment by ghost** [comment]
Anyone else has any ideas for how to use sonic the hedgehog sprite, lay em on me.

**Comment by Manly-Dwarf** [comment]
Attack it with red shoes/a chili dog to make it into Sonic instead.

**Comment by ghost** [comment]
Do chili dogs even exist? The red shoes thing sounds good.

**Comment by ghost** [comment]
+1

File: hippiestation.dme [hunk 1]
```diff
@@ -691,10 +691,10 @@
 #include "code\game\objects\effects\decals\decal.dm"
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\cleanable\humans.dm"
-#include "code\game\objects\effects\decals\cleanable\misc.dm"
-#include "code\game\objects\effects\decals\cleanable\robots.dm"
+#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
+#include "code\game\objects\effects\decals\Cleanable\humans.dm"
```
Comment (Carbonhell) [inline hippiestation.dme:695]: update your master,seems outdated

**Comment by Carbonhell** [comment]
is this ready? also iunno, current effects looks boring af,-1

File: hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm [hunk 2]
```diff
@@ -54,4 +59,14 @@
 		scared = TRUE
 		scared_time = world.time + FEAR_DURATION
 
+/mob/living/simple_animal/pet/hedgehog/emag_act(mob/user)
+	if(emagged == FALSE)
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm:63]: if(!emagged)

File: hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm [hunk 3]
```diff
@@ -54,4 +59,14 @@
...
+		emagged = TRUE
+		do_sparks(8, FALSE, loc)
+		icon_state = "Sonic"
+		icon_living = "Sonic"
+		icon_dead = "Sonic_dead"
+		turns_per_move = 1
+		visible_message("<span class = 'warning'> [src] starts to go fast! </span>")
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm:70]: Remove space before and after the message itself

**Comment by GuyonBroadway** [comment]
@Spankmonkey777 Hurry up and make the changes John and Bawb requested.

**Comment by praisenarsie** [comment]
staler than dirt

@Spankmonkey777 ping for reopen

**Pull request closed without merging**